dll/win32/shell32/CDefView.cpp

```diff
@@ -607,11 +614,39 @@ void CDefView::UpdateStatusbar()
             pIcon = (LPARAM)m_hMyComputerIcon;
         }
 
+        if (hThread != m_hUpdateStatusbarThread)
```

Comment:
If the target variable has changed In the iteration of selection items, it might be not updated.
If it was outside of the iteration, it might be updated.

Reply:
Refactored and optimized.